proposals/ref-out-lambda-params.md

```diff
@@ -23,17 +23,39 @@ TryParse<int> parse2 = (string text, out int result) => Int32.TryParse(text, out
 
 ### Parameter declaration
 
-Parameter declarations in lambda expressions now permit a single identifier after an `in`/`ref`/`out` modifier on the parameter.
+Parameter declarations in lambda expressions with parenthesized parameters now permit a single identifier after an `in`/`ref`/`out` modifier on the parameter. This does not apply to lambda expressions with a single parameter with omitted parentheses.
```

Comment:
Note. We may have to also support `ref readonly ` as that feature is currently being worked on. If that impacts delegates, it would impact this.  @jjonescz ?

Reply:
So do we also declare the support for `ref readonly` in the spec before its approval?

Reply:
Ref-readonly is approved and shipping for 12.  Subsequent proposals should incorporate the fallout from them. :-)